client/src/axios-auth.js

```diff
@@ -89,7 +89,9 @@ instance.interceptors.response.use(function (response) {
     $(error.config.spinner).addClass('hidden')
   }
 
-  if (error.request && error.request.url.includes('/api/v1/events/')) return Promise.reject(error)
```

Comment:
`url` doesn't exist on `request`. But `responseURL` does.

Reply:
@kialj876 Does checking for this path make sense?

Reply:
(Whether this condition passes or not, the promise is rejected, but possibly with a `checkError` to commit the response JSON/warnings/errors/message. I don't know what is done with that afterwards.)

Reply:
yeah that's a good question. I don't think it's necessary, but hard to be sure since this could be for an edge case we've forgotten

Reply:
I'm thinking it probably puts up a modal that shows an error msg that would be incorrect for the events endpoint?

Reply:
Thanks.